datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeAdapter043CompatTest.java

```diff
@@ -67,8 +67,10 @@ void writeNewAndReadFileFromOldVersion() {
         // validate folder structure
         assertThat(Dirs.walk(dfsRoot, 1)).containsExactlyInAnyOrder("profiles", "users");
         assertThat(Dirs.walk(dfsRoot.resolve("profiles")))
+                .extracting(string -> string.replaceAll("\\\\","/"))
```

Comment:
Better to use URI form

Reply:
How would that look like?

Reply:
Actually replaceAll seem to be useless here